### PR TITLE
QA-453

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

File: checklinks.js [hunk 1]
```diff
@@ -17,7 +15,6 @@ engine(
     extensions: ['md'],
     pluginPrefix: 'remark',
     packageField: 'remarkConfig',
-    injectedPlugins: [parse, stringify],
```
Comment (oleorhagen) [inline checklinks.js]: why is this not needed anymore?

**Review by oleorhagen — APPROVED** [review]
Thanks!

File: checklinks.js (hunk 1 above)
Comment (mzedel) [reply checklinks.js]: because of the `pluginPrefix` defined above - any package with the _remark_ prefix is considered a plugin 👌

Comment (oleorhagen) [reply checklinks.js]: Nice!

**Comment by mzedel** [comment]
@mender-test-bot cherry-pick to:
 * 3.4.x
 * 3.3.x

**Pull request merged**